db/migrate/20160215223349_create_picks.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePicks < ActiveRecord::Migration
+  def change
+    create_table :picks do |t|
+      t.belongs_to :user, null: false
```

Comment:
A shortcut here is:

```ruby
t.belongs_to :user, null: false, index: true
```

What you did is fine though. Maybe even better as it's very clear what is being added to the database. Shortcut for your knowledge.